Apparently no one tried building this with clutter-win32 in a *long* time

--- origsrc/clutter-gtk-0.10.8/configure.ac	2012-05-04 15:56:00.974731000 -0500
+++ src/clutter-gtk-0.10.8/configure.ac	2012-05-04 14:57:27.000000000 -0500
@@ -76,10 +76,10 @@ AS_CASE([$with_flavour],
 
         [win32], [
           AC_DEFINE([HAVE_CLUTTER_GTK_WIN32], 1, [Using the Win32 flavour])
-          PKG_CHECK_MODULES(CLUTTER, clutter-win32-1.0 >= $CLUTTER_REQUIRED)
+          PKG_CHECK_MODULES(CLUTTER, clutter-win32-1.0 >= clutter_req_version)
           PKG_CHECK_MODULES(GTK, gtk+-win32-2.0 >= gtk_req_version)
           CLUTTER_GTK_REQUIRES="clutter-win32-1.0 gtk+-win32-2.0"
-        ]
+        ],
 
         [AC_MSG_ERROR([Invalid argument for --with-flavour])]
 )
--- origsrc/clutter-gtk-0.10.8/clutter-gtk/gtk-clutter-embed.c	2010-09-16 07:49:52.000000000 -0500
+++ src/clutter-gtk-0.10.8/clutter-gtk/gtk-clutter-embed.c	2014-01-28 22:39:45.662204600 -0600
@@ -53,7 +53,7 @@
 
 #elif defined(HAVE_CLUTTER_GTK_WIN32)
 
-#include <clutter/clutter-win32.h>
+#include <clutter/win32/clutter-win32.h>
 #include <gdk/gdkwin32.h>
 
 #endif /* HAVE_CLUTTER_GTK_{X11,WIN32} */
--- origsrc/clutter-gtk-0.10.8/clutter-gtk/gtk-clutter-util.c	2010-09-16 08:46:51.000000000 -0500
+++ src/clutter-gtk-0.10.8/clutter-gtk/gtk-clutter-util.c	2014-01-28 22:39:54.812727900 -0600
@@ -16,7 +16,7 @@
 
 #elif defined(HAVE_CLUTTER_GTK_WIN32)
 
-#include <clutter/clutter-win32.h>
+#include <clutter/win32/clutter-win32.h>
 #include <gdk/gdkwin32.h>
 
 #endif /* HAVE_CLUTTER_GTK_{X11,WIN32} */
